### ISSUE TITLE
Content transforms

### ISSUE THREAD

**Issue by taylortom** [taylortom/at-migrate#5]
The following should be fixed from exports from legacy:

### config.json
- ~`_accessibility._ariaLevels._menuGroup` => missing, should be `2`~ _fixed in schema_

### contentobjects.json
- `_vanilla._backgroundImage` => no `""`
- `_vanilla._backgroundStyles` => no `null`
- `_vanilla._pageHeader._backgroundImage` => no `""`
- `_vanilla._pageHeader._backgroundStyles` => no `null`

**Comment by taylortom** [comment]
Done
